### PR TITLE
completed assessment

### PR DESCRIPTION
**Pull request by Aizada-mollenkopf**
assessment#5 ruby

### CODE REVIEW

File: ruby.md [hunk 1]
```diff
@@ -5,41 +5,67 @@ Answer the following questions. First, without external resources. Challenge you
...
+   class Body
+   has_many:hands
+   end
+   class Hand
+   has_many:fingers
+   end
+   class Finger
+   belongs_to:hands
+   end
+
```
Comment (sjproctor) [inline ruby.md:60]: Nice!

File: ruby.md [hunk 2]
```diff
@@ -51,15 +77,40 @@ Answer the following questions. First, without external resources. Challenge you
...
+  Researched answer: In Ruby, blocks are snippets of code that can be created to be executed later. 
+  Blocks are passed to methods that yield them within the do and end keywords. 
+  One of the many examples is the #each method, which loops over enumerable objects.
+  A “proc” is an instance of the Proc class, which holds a code block to be executed, and can be stored in a variable. 
+  To create a proc, you call Proc.new and pass it a block.
+  Lambdas are essentially procs with some distinguishing factors. They are more like “regular” methods in two ways: 
+  they enforce the number of arguments passed when they’re called and they use “normal” returns.
+  
+
```
Comment (sjproctor) [inline ruby.md:116]: Good job!

**Review by sjproctor — APPROVED** [review]
Nice work! Good job putting in a lot of work!

**Comment by Aizada-mollenkopf** [comment]
thank you!

**Pull request merged**